task3/script.js

```diff
@@ -0,0 +1,17 @@
+const ul = document.getElementById('ul');
+const liLength = 2;
+const data = ["bookmark.png", "message.png"];
+
+for (let i =1; i <= liLength; i++) {
```

Comment:
```suggestion

const data = ["bookmark.png", "message.png"];

for (let i =1; i <= data.length; i++) {
```
[nits]
You can use .length to get the number of elements in the array.

Reply:
I see, the number of arrays and the number of variables are the same.

Thank you for the review.

Reply:
Thanks for the fix.
You can read more about it [this](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Array/length)